### PR TITLE
Restore ability to theme the menu applet menu separately from other m…

### LINKED ISSUE #5934: Main menu background no longer themable [3.2.1]?
Using Cinnamon 3.2.1 on Arch. It seems it is no longer possible to separately theme the main menu applet's background using .menu-background; instead it takes the theme of .menu like all the other applet popup menus. This seems like a bug as the .menu-background style class is still in menu@cinnamon.org/applet.js, but for some reason is being ignored. This could break many 3rd party themes that attempt to theme the main menu differently than the applet popup menus.

Downgrading to cinnamon 3.0.x resolves the problem. Looking Glass inspector on 3.2.1 shows "StBin.menu bottom" instead of "StBin.popup-menu-boxpointer popup-menu menu-background" on 3.0.x. Perhaps .menu-background isn't being properly applied now that .popup-menu-boxpointer is gone in 3.2.x?

### PR DESCRIPTION
**Pull request by collinss**
…enus

Fixes #5934

### CODE REVIEW

File: files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js [hunk 1]
```diff
@@ -1150,7 +1150,7 @@ MyApplet.prototype = {
         this.settings.bind("activate-on-hover", "activateOnHover", this._updateActivateOnHover);
         this._updateActivateOnHover();
 
-        this.menu.actor.add_style_class_name('menu-background');
+        this.menu.setCustomStyleClass('menu-background');
```
Comment (lots0logs) [inline files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js]: @collinss It'd be nice if all applets added a predictable custom style class to their menus. It won't happen overnight but I think this PR is a good place to start. What do you think?
```javascript
  this.menu.setCustomStyleClass('menu-background menu-for-applet--menu');
```

Then eventually...
```javascript
 /* Window List Applet */
  this.rightClickMenu.setCustomStyleClass('menu-for-applet--windowlist');
```
```javascript
 /* Sound Applet */
  this.menu.setCustomStyleClass('menu-for-applet--sound');
```

Comment (lots0logs) [inline files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js]: @collinss 
> In fact, we want all the other menus to be uniform in appearance. 

Sorry that doesn't make much sense to me. I mean why even have themes in the first place if you don't allow them to use whatever styles their designers want to use? Consistency is important, all of the official Cinnamon/Linux Mint themes should definitely be consistent. That can easily be achieved while at the same time allowing unofficial theme authors to be as creative as they like.  

The bottom line is that implementing what I described costs the project absolutely nothing thanks to the work you already did on this PR. That extra style class won't change anything for users of themes that don't utilize that class.

Comment (zagortenay333) [inline files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js]: Don't applets already have the ability to do this?

The weather applet does (because I requested it) and some of your applets @collinss do too (also because I opened a request.)

In many applets one can utilize this for some really cool effects.

**Comment by Odyseus** [comment]
Hello, everybody.

@collinss: there is another instance of `this.menu.actor.add_style_class_name('menu-background');` inside the **on_orientation_changed** method. Shouldn't the `setCustomStyleClass` method be used in there too? I'm just pointing that out because I'm not sure if not using `setCustomStyleClass` would override all other classes the menu could have.
